### PR TITLE
Support lowercase viewbox attribute

### LINKED ISSUE #355: Fails on lowercase `viewbox` property
While technically not a full SVG, browsers can still read this, but it fails in `image-size`

```js
var imageSize = require("image-size")
imageSize(Buffer.from('<svg viewbox="0 0 10 10"></svg>', 'utf8'))

// -> TypeError: Invalid SVG
```

```js
var imageSize = require("image-size")
imageSize(Buffer.from('<svg viewBox="0 0 10 10"></svg>', 'utf8'))

// -> Object {height: 10, width: 10, type: "svg"}
```

Tested on https://npm.runkit.com/image-size

### PR DESCRIPTION
**Pull request by fregante**
Fixes #355

### CODE REVIEW

File: lib/types/svg.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ const svgReg = /<svg\s([^>"']|"[^"]*"|'[^']*')*>/
 const extractorRegExps = {
   height: /\sheight=(['"])([^%]+?)\1/,
   root: svgReg,
-  viewbox: /\sviewBox=(['"])(.+?)\1/,
+  viewbox: /\sview[bB]ox=(['"])(.+?)\1/,
```
Comment (fregante) [inline lib/types/svg.ts:14]: Alternatively it can be case-insensitive, like in browsers
```suggestion
  viewbox: /\sviewBox=(['"])(.+?)\1/i,
```

**Comment by netroy** [comment]
@fregante thanks for the PR. Can you please add a test as well?

**Comment by fregante** [comment]
I can see it being picked up now 👍 

```
    specs/images/valid/svg/viewbox-height.svg
      ✓ should return correct size for specs/images/valid/svg/viewbox-height.svg
    specs/images/valid/svg/viewbox-lowercase.svg
      ✓ should return correct size for specs/images/valid/svg/viewbox-lowercase.svg
    specs/images/valid/svg/viewbox-units.svg
      ✓ should return correct size for specs/images/valid/svg/viewbox-units.svg
```

**Comment by netroy** [comment]
Thanks. Published as `v0.9.5`

**Pull request merged**